ktext.md

```diff
@@ -353,25 +353,29 @@ To figure out what kind of keyboard and trackpad you have, check Device Manager
 
 * [VoodooPS2](https://github.com/acidanthera/VoodooPS2/releases)
   * For systems with PS2 keyboards, mice, and trackpads
-  * Requires OS X 10.11 or newer for MT2 (Magic Trackpad 2) functions
+  * Requires macOS 10.11 or newer for MT2 (Magic Trackpad 2) functions
 * [RehabMan's VoodooPS2](https://github.com/RehabMan/OS-X-Voodoo-PS2-Controller)
-  * For older systems with PS2 keyboards, mice, and trackpads
-  * Use if you need OS X 10.6+ support, or otherwise don't want to use VoodooInput for MT2 functions
+  * For older systems with PS2 keyboards, mice, and trackpads; or when you don't want to use VoodooInput
+  * Supports macOS 10.6+ support
 * [VoodooRMI](https://github.com/VoodooSMBus/VoodooRMI/releases/)
   * For systems with Synaptics' SMBus-based devices, mainly for trackpads and trackpoints.
```

Comment:
```suggestion
  * For systems with Synaptics's SMBus-based devices, mainly for trackpads and trackpoints.
```

Reply:
Synaptics' is correct since it ends in 's' already.